src/main/java/organice/logic/parser/FuzzyFindCommandParser.java

```diff
@@ -0,0 +1,44 @@
+package organice.logic.parser;
+
+import static organice.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static organice.logic.parser.CliSyntax.PREFIX_AGE;
+import static organice.logic.parser.CliSyntax.PREFIX_BLOOD_TYPE;
+import static organice.logic.parser.CliSyntax.PREFIX_DOCTOR_IN_CHARGE;
+import static organice.logic.parser.CliSyntax.PREFIX_NAME;
+import static organice.logic.parser.CliSyntax.PREFIX_NRIC;
+import static organice.logic.parser.CliSyntax.PREFIX_ORGAN;
+import static organice.logic.parser.CliSyntax.PREFIX_ORGAN_EXPIRY_DATE;
+import static organice.logic.parser.CliSyntax.PREFIX_PHONE;
+import static organice.logic.parser.CliSyntax.PREFIX_PRIORITY;
+import static organice.logic.parser.CliSyntax.PREFIX_TISSUE_TYPE;
+import static organice.logic.parser.CliSyntax.PREFIX_TYPE;
+
+import organice.logic.commands.FuzzyFindCommand;
+import organice.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new FindCommand object
+ */
+public class FuzzyFindCommandParser implements Parser<FuzzyFindCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the FindCommand
+     * and returns a FindCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public FuzzyFindCommand parse(String args) throws ParseException {
+        args = args.replaceAll("\n", " ").replaceAll("\\s+", " ");
+        String trimmedArgs = args.trim();
+        ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args, PREFIX_NAME, PREFIX_NRIC, PREFIX_PHONE,
+                PREFIX_TYPE, PREFIX_AGE, PREFIX_PRIORITY, PREFIX_BLOOD_TYPE, PREFIX_DOCTOR_IN_CHARGE,
+                PREFIX_TISSUE_TYPE, PREFIX_ORGAN_EXPIRY_DATE, PREFIX_ORGAN);
+
+        // Preamble length > 7 indicates initial command is not `fuzfind PREFIX/KEYWORD`
```

Comment:
May I ask, what is the purpose of this line of code? If I'm not wrong, you could declare a String constant `COMMAND_WORD` in the `FuzzyFindCommand` class and use that.